search/includes/classes/class-queue.php

```diff
@@ -596,6 +599,26 @@ public function get_jobs( $job_ids ) {
 		return $jobs;
 	}
 
+	/**
+	 * Get the jobs using the lowest and highest job_id range
+	 */
+	public function get_jobs_by_range( $min_id, $max_id ) {
+		global $wpdb;
+
+		if ( ! $min_id || ! $max_id ) {
+			return array();
+		}
+
+		$table_name = $this->schema->get_table_name();
+
+		$min_job_id_value = intval( $min_id );
+		$max_job_id_value = intval( $max_id );
+
+		$jobs = $wpdb->get_results( "SELECT * FROM {$table_name} WHERE `job_id` >= $min_job_id_value AND `job_id` <= $max_job_id_value" ); // Cannot prepare table name, ids already escaped. @codingStandardsIgnoreLine
```

Comment:
Looks like the comment on the #L641 is not accurate and the `$wpdb->prepare` is actually used? If that's the case, let's also use it here and remove the comment.

Reply:
I added the prepare statement. However the issue both here and #L641 has in common is that you can't prepare table name.

So you can not do `$wpdb->prepare("SELECT * FROM %s", "my-table")`. However if you inline the table name as here the coding standard complains. Hence the ignore statement and the comment

Reply:
ah I see, I misread the comment as "can't prepare table_name so won't be preparing at all"